Sanpham.java

```diff
@@ -0,0 +1,39 @@
+import java.util.Scanner;
+
+public class Sanpham{
+    public String tenSp;
+    public double donGia;
+    public double giamGia;
+    public Sanpham(){
+    }
+    public void nhap(){
+        Scanner sc = new Scanner(System.in);
+        System.out.print("Nhap ten san pham: ");
+        tenSp = sc.next();
+        System.out.print("Nhap don gia: ");
+        donGia = sc.nextDouble();
+        System.out.print("Nhap giam gia: ");
+        giamGia = sc.nextDouble();
+    }
+    private double ThueNhapKhau(){

```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.